packages/expo-linear-gradient/src/LinearGradient.tsx

```diff
@@ -58,14 +58,20 @@ export type LinearGradientProps = ViewProps & {
    * @default { x: 0.5, y: 1.0 }
    */
   end?: LinearGradientPoint | null;
+
+  /**
+   * Enables or disables paint dithering on Android. Paint dithering can fix color banding issue.
+   * @default false
+   */
+  ditherPaintAndroid?: boolean | false;
```

Comment:
```suggestion

  /**
   * Enables or disables paint dithering on Android. Paint dithering can fix color banding issue.
   * @default false
   * @platform android
   */
  ditherPaint?: boolean;
```

Typing, naming and JSDoc nit. We usually do not include the platform name in platform specific props..